vendors/pc/boards/windows/aws_tests/application_code/cellular_setup.c

```diff
@@ -99,14 +94,14 @@
                     ( ( simStatus.simCardState == CELLULAR_SIM_CARD_INSERTED ) &&
                       ( simStatus.simCardLockState == CELLULAR_SIM_CARD_READY ) ) )
                 {
-                    FreeRTOS_printf( ( ">>>  Cellular SIM okay  <<<\r\n" ) );
+                    configPRINTF( ( ">>>  Cellular SIM okay  <<<\r\n" ) );
                     break;
                 }
                 else
                 {
-                    FreeRTOS_printf( ( ">>>  Cellular SIM card state %d, Lock State  <<<\r\n",
-                                       simStatus.simCardState,
-                                       simStatus.simCardLockState ) );
+                    configPRINTF( ( ">>>  Cellular SIM card state %d, Lock State  <<<\r\n",
```

Comment:
In this log, there is only one %d, but having two parameters.

Reply:
This will be updated in next version.